Makefile

```diff
@@ -27,4 +27,27 @@ all-tests:
 generate build clean:
 	go $@ $(EXTRAGOARGS)
 
-.PHONY: all generate clean build test unit-tests all-tests
+clean:
+	go $@ $(EXTRAGOARGS)
+	# --force to prevent erroring on stamp file not existing
+	rm --force sandbox-test-fc-build-stamp
```

Comment:
You can also write `rm sandbox-test-fc-build-stamp ||:` prevent an error.  `||` is a logical or and `:` is a builtin for true.